.github/workflows/ci_release.yml

```diff
@@ -0,0 +1,69 @@
+name: Release
+on:
+  push:
+  schedule:
+    # Run daily at 1:15am
+    - cron: "15 1 * * *"
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    inputs:
+      version:
+        # Friendly description to be shown in the UI instead of 'name'
+        description: "Semver type of new version (major / minor / patch)"
+        # Input has to be provided for the workflow to run
+        required: true
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
```

Comment:
Is this input used anywhere?

Reply:
These are options that you as a user will be able to select when triggering the workflow manually to create a release.

Reply:
That much I got. 😅 What is the `version` used for in this workflow?

Reply:
Oh i see the error. fixed.